metric-example/metric-expressions-for-k8s.md

```diff
@@ -79,9 +81,9 @@ builtin:host.cpu.usage:avg
 By replacing the *usage* part of this query, we can solve the same use cases for **requests** and **limits**. For example, for **requests** the last metric expression above would result in the following query:
 ```
 (
-  (builtin:cloud.kubernetes.node.cpuRequested:avg)
-  /(builtin:cloud.kubernetes.node.cores:avg)
-  *(100)
+  builtin:cloud.kubernetes.node.cpuRequested:avg
+  /builtin:cloud.kubernetes.node.cores:avg
```

Comment:
I would also use white spaces in this example.

Reply:
thx - done